chart/templates/deployment-driver.yaml

```
@@ -25,6 +25,9 @@ spec:
           command:
           - longhorn-manager
           - -d
+          {{- if eq .Values.longhornManager.log.format "json" }}
```

Do you mean `longhornDriver.log.format` ?

```suggestion
          {{- if eq .Values.longhornDriver.log.format "json" }}
```

---

Yes, sorry, i fixed it